app/services/document_converter_service.rb

```diff
@@ -0,0 +1,42 @@
+class DocumentConverterService
```

Comment:
a service typically just responds to `call` - this has not been adhered to very well previously either.